src/registry/addressNL/edit.tsx

```diff
@@ -45,6 +46,26 @@ const DeriveAddress = () => {
   );
 };
 
+const ColumnsLayout: React.FC = () => (
+  <>
+    <Select
+      name="layout.show"
```

Comment:
```suggestion
      name="layout"
```
Can you tell me why you think you need to include a `.show` nested property in there? I've seen it everywhere except in the type definition now :thinking: